src/components/__tests__/ars-test.jsx

```diff
@@ -13,11 +13,16 @@ describe("Ars", function() {
 
   describe("when the component renders", function() {
     let onChange  = sinon.spy()
-    let component = Test.renderIntoDocument(makeComponent({ onChange }))
+    let picked = 9
+    let component = Test.renderIntoDocument(makeComponent({ onChange, picked }))
 
     it ("has a selection component", function() {
       component.refs.should.have.property("selection")
     })
+
+    it ("migrates a single `picked` value to an array", function() {
+      component.state.should.have.property("picked").that.deep.equals([9])
```

Comment:
Oh right, I like using `that` here. I found that `deep` was necessary on arrays in some earlier testing. When I used `eql` I got messages like `Expected [ 0 ] to be equal to [ 0 ]`.